src/main/java/dtb/lego/truck/rest/component/events/boundary/ComponentsController.java

```diff
@@ -0,0 +1,50 @@
+package dtb.lego.truck.rest.component.events.boundary;
```

Comment:
There is no reason to have the events package here

Reply:
Do you think it makes sense to move everything regarding the Events out from the Components package and unify everything from the Events and DataAcquisition into a single package?